docs/Configuration/SSO.md

```diff
@@ -21,6 +21,18 @@ NTLM SSO allows for users to be logged in automatically if they are already logg
 
 Check __Services -> Properties -> SSO__ (Windows Kerberos/NTLM)
 
+### Windows Based SSO (NTLM): Automatic Logon settings
+If SSO is not functioning as expected (e.g. the browser displays a Windows Security authentication window prompting for credentials), check the settings in Internet Explorer around Automatic Logon:
```

Comment:
This is too detailed and targeting an end user - it t should target an administrator